debezium-api/src/main/java/io/debezium/jdbc/DecimalMode.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.jdbc;
```

Comment:
I had to extract this out since this was user all over the shop (incl. in some of the logical types). Seems common enough that it deserves to be a first-class enum in the API. Some other things from the JDBC package can be moved, but I didn't want to disturb too much.